src/Tribe/Migration/Queue_4_12.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * Queue to handle Attendees Block REST migration.
+ *
+ * @since TBD
+ */
+
+namespace Tribe\Tickets\Migration;
+
+use Tribe\Tickets\Events\Attendees_List;
+use Tribe__Tickets__Main;
+use WP_Query;
+
+class Queue_4_12 extends Queue {
+
+	/**
+	 * {@inheritDoc}
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public $queue_id = '4_12';
+
+	/**
+	 * {@inheritDoc}
+	 *
+	 * @since TBD
+	 *
+	 * @param WP_Post|int $post The post object or post ID, depending on what $this->get_query() is set to return.
+	 *
+	 * @return bool Whether the post was processed.
+	 */
+	public function process_post( $post ) {
+		$has_attendee_list_shortcode = has_shortcode( $post->post_content, 'tribe_attendees_list' );
+		$has_attendee_list_block     = function_exists( 'has_block' ) ? has_block( 'tribe/attendees', $post ) : false;
+
+		$has_attendee_list = $has_attendee_list_shortcode || $has_attendee_list_block;
+
+		update_post_meta( $post->ID, Attendees_List::HIDE_META_KEY, (int) $has_attendee_list );
+
+		return true;
+	}
+
+	/**
+	 * {@inheritDoc}
+	 *
+	 * @since TBD
+	 *
+	 * @param int $current_offset The current offset being used.
+	 *
+	 * @return int[]|WP_Post[] List of post IDs or post objects.
+	 */
+	public function get_posts( $current_offset ) {
+		$supported_post_types = Tribe__Tickets__Main::instance()->post_types();
+
+		if ( empty( $supported_post_types ) ) {
+			return [];
+		}
+
+		$hide_meta_key = Attendees_List::HIDE_META_KEY;
+
+		$searches = [
+			'[tribe_attendees_list',
+			'<!-- wp:tribe/attendees ',
+		];
+
+		$args = [
+			'post_type'      => $supported_post_types,
+			'post_status'    => 'any',
+			'posts_per_page' => $this->batch_size,
+			'offset'         => $current_offset,
+			'orderby'        => 'date',
+			'order'          => 'ASC',
+			'meta_query'     => [
+				'operator' => 'OR',
```

Comment:
:warning: Detected usage of meta_query, possible slow query.

<sub>hash: 39cf778a080c9c550a11d794a496c45c</sub>